bubbly-bg/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "bubble-bg",
+    "filename": "bubbly-bg.min.js",
+    "description": "Lightweight and beautiful bubbly backgrounds",
+    "author": "David Åse",
+    "license": "Apache-2.0",
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/tipsy/bubbly-bg.git"
+    },
+    "homepage": "https://github.com/tipsy/bubbly-bg#readme",
```

Comment:
Please use `https://tipsy.github.io/bubbly-bg/` as the homepage instead. Because it have a link to its repo.